include/braincloud/reason_codes.h

```diff
@@ -317,4 +317,200 @@ static const int CLIENT_UPLOAD_FILE_TIMED_OUT = 90101;
 static const int CLIENT_UPLOAD_FILE_UNKNOWN = 90102;
 static const int CLIENT_DISABLED = 90200;
 
+static const int RECREATING_ANONYMOUS_MEMORY = 40203;
+
```

Comment:
I know all the extra new line here is weird, but is it really important? The grouping above doesn't really make any sense or follow any logic anyway.

This is a side effect of the auto generation...

Reply:
Seems fine to me. Good to hear your auto-generating them